src/drivers/glfw/lv_opengles_driver.c

```diff
@@ -100,10 +109,17 @@ static const char * fragment_shader =
     "uniform sampler2D u_Texture;\n"
     "uniform int u_ColorDepth;\n"
     "uniform float u_Opa;\n"
+    "uniform bool u_IsFill;\n"
+    "uniform vec3 u_FillColor;\n"
     "\n"
     "void main()\n"
```

Comment:
I went over the shader as well and see that precision is set to mediump. That is fine for texture coordinates, but the use only seems to be limited to colors. you can use lowp. In large scenes using framents shaders with lowp, midiump or higp can have some influence on performance (but probably not really needed in this case)